packages/prng/unix_ts_rng.go

```diff
@@ -4,6 +4,8 @@ import (
 	"bytes"
 	"encoding/binary"
 	"time"
+
+	"github.com/dchest/blake2b"
```

Comment:
Why aren't we using the default `"golang.org/x/crypto/blake2b"`?

Reply:
Now we don't need that dependency anymore :P